book/languages/js/intro.md

```diff
@@ -0,0 +1,44 @@
+# Introduction
+
+## Inception
+
+**JavaScript (JS)**, but the official name is **ECMAScript**. <br> This language is really fast scripting language using specially for web pages, for **frontend** (Vue.js, React.js etc.) and for **backend** (node.js etc.). It can be used also for creating desktop and mobile native apps (Proton, Electron).
+**Brednan Eich** created that language, and it firstly appeared in **December 4, 1995**. JS became an **ECMA standard in 1997**. <br> File extension is **.js**
+
+## Trademark
+
+Actually Oracle have a trademark to JavaScript. Yes, these guys from Silicon Valley that are making virtual machines software.
+
+## Versions
+
+| **Year**      | **Name**      | **Description** |
+| ------------- | ------------- | ----- |
+| 1997          | ECMAScript 1  | First Edition |
+| 1998          | ECMAScript 2  | Editorial changes only |
+| 1999          | ECMAScript 3  | Added Regular Expressions, Added try/catch |
+|               | ECMAScript 4  | Was never released |
+| 2009          | ECMAScript 5  | Added "strict mode". Added JSON support. |
+| 2011          | ECMAScript 5.1| Editorial changes |
+| 2015          | ECMAScript 6  | Many new features (including let, const and default parameter values) |
+| 2016          | ECMAScript 7  | Added exponential operator (**), Added Array.prototype.includes  |
+| 2017          | ECMAScript 8  | Added string padding. Added new Object properies. Added Async functions. Added Shared Memory. |
+
+## Why you should use JavaScript?
+
+**You should use JavaScript** because:
+
+- Many of people actually use it, big community,
+- It is functionally in most of the browsers,
+- If you'll work as JS dev, you will be high paid one,
+- Many job offers in which JS is 'must have',
+- It is fast language,
+- It have many frameworks, I mean, **many frameworks**,
+- You could use it wherewhere you want.
+
+## Why you shouldn't use JavaScript?
+
+**You shouldn't use JavaScript** because:
+
+- It is fast, but not as fast as C or Rust,
+- Many of people know it, it is good, but for searching one ideal job as a JS dev, it isn't,
+- It is nice to have a lot of frameworks, for backend and frontend, but the way, how much frameworks are for JS became a meme.
```

Comment:
Add a text

Reply:
Try to add the security aspect of JS and why it is a potential threat to website security

Reply:
That isn't true, you know ?

Reply:
@FilipeRamalho I beg to differ. Though JS has a strict mode it is not that widely implemented and JS can be used in attacking a client of a badly coded website. Though not the fault of JS it is related neverthless. 
https://stackoverflow.com/questions/3793246/javascript-security-risks

Reply:
Everything as risks, but that isn't a disadvantage. A disadvantage would be a security risk that is part of Javascript, but these are general web developement risks.